packages/api-specification/interface/screen.ts

```diff
@@ -0,0 +1,36 @@
+declare namespace ssf {
+  export abstract class Screen {
+    static getDisplays(): Promise<Array<Display>>;
+  }
+
+  /**
+   * Information about the users display. Note that the Browser API does not support multiple
+   * displays, so it assumes the display the browser is running in is the only display.
+   */
+  export interface Display {
+    /**
+     * Unique Id of the display.
+     */
+    id: string;
```

Comment:
That's not something that is supported naively, although we could add that into our API. To do that currently, you would have to position the window within the bounds of the 2nd window (i.e. `x: 1920, y:0`)